# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libtirpc/no-gss.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

if gss is disabled, then svc_auth_gss_ops won't be available

--- ./src/svc.c.orig
+++ ./src/svc.c
@@ -77,7 +77,7 @@

 extern rwlock_t svc_lock;
 extern rwlock_t svc_fd_lock;
-extern struct svc_auth_ops svc_auth_gss_ops;
+extern struct svc_auth_ops svc_auth_gss_ops __attribute__((weak));

 static struct svc_callout *svc_find (rpcprog_t, rpcvers_t,
                                     struct svc_callout **, char *);
